src/components/FeedLikesTweet.tsx

```diff
@@ -187,7 +187,7 @@ const FeedLikesTweet = ({ tweet, user }: Like) => {
               <div className="px-1">{tweet.stats.comments}</div>
             </StatisticOfTweet>
             <RetweetButton retweetCount={tweet.stats.retweets} />
-            <LikeButton likesCount={tweet.stats.likes} />
+            <LikeButton likesCount={tweet.stats.likes} id={tweet._id} />
```

Comment:
LikeButton should receive only 3 props:
- likesCount
- isLiked
- onLikeClick

because it's dumb component, and it shouldn't contain any logic to send data to backend. Instead it should be comepleted from parent components, like FeedLikesTweet, FeedSingleTweet